docs/change-log/_index.md

```diff
@@ -0,0 +1,339 @@
+(change-log)=
+# Change log
+- [v1.2.1](#v1-2-1)
+- [v1.2.0](#v1-2-0)
+- [v1.1.3](#1-1-3)
+- [v1.0.6](#v1-0-6)
+- [v1.0.5](#v1-0-5)
+- [v1.0.4](#v1-0-4)
+- [v1.0.3](#v1-0-3)
+- [v1.0.2](#v1-0-2)
+- [v1.0.0](#v1-0-0)
+- [Open issues](#open-issues)
+- [Limitations](#limitations)
+- [Deprecations](#deprecations)
+
+## v1.2.1
+
+### New and updated features
+
+#### Feature store
+- Supports ingesting Avro-encoded Kafka records. [View in Git](https://github.com/mlrun/mlrun/issues/2649).
+
+### Closed issues
+- When running a workflow with `local=True`, the serving function is updated. [View in Git](https://github.com/mlrun/mlrun/issues/2716).
+- The Kubernetes **Pods** tab in **Monitor Workflows** now shows the complete pod details. [View in Git](https://github.com/mlrun/mlrun/pull/1576).
+- When selecting **All** in the **Tag** dropdown list of the **Projects | Feature Store | Feature Vectors** tab, 
+all of the vectors display correctly. [View in Git](https://github.com/mlrun/ui/pull/1549).
+- MLRun API returns the correct `next_run_time` value. [View in Git](https://github.com/mlrun/mlrun/pull/2862).
+- Filtering in **Projects | Jobs | Monitor Workflows** returns correct results for all ranges. (Previously was not accurate 
+for > 1 hour.) [View in Git](https://github.com/mlrun/mlrun/pull/2786).
+- The updated tooltips in **Projects | Jobs | Schedule** explain that day 0 (for cron jobs) is Monday, and not Sunday. 
+[View in Git](https://github.com/mlrun/ui/pull/1571).
+- When creating a project, the error `NotImplementedError` was updated to explain that MLRun does not have a 
+DB to connect to. [View in Git](https://github.com/mlrun/mlrun/pull/2856).
+- Previewing artifacts in the UI was fixed. [View in Git](https://github.com/mlrun/mlrun/pull/2592).
```

Comment:
Its hard to understand what is being fixed, e.g I would change 
MLRun API returns the correct `next_run_time` value -> Fix not updating `next_run_time` when skipping scheduling due to concurrent runs

In my opinion this way we explain what was fixed and not what is working.